.eslintrc.js

```diff
@@ -7,14 +7,14 @@ module.exports = {
     },
```

Comment:
For the eslintrc environment, I had to add browser: true. Maybe remove node: true here, and add browser: true because of the override in the webpack config.

Reply:
that seems to be done further down in the `overrides` section. 

I do need to move the `prettier/recommended` plugin to the end of the array though